.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.8'
+          - '1.9'
```

Comment:
I'm reluctant to test a Julia version that Oceananigans isn't tested against yet, and not test the version that Oceananigans _is_ tested agaisnt.

Reply:
Damn! That was a mistake! I was meant to do that for a different repo and did it here apparently... I'll remove this! :)